### DISCUSSION TITLE
Execution sequence when plugin conflicts

### DISCUSSION THREAD

**Discussion by 1798972** [Kong/kong#8944] [Help] [answered]
If I apply the same plug-in to both service and route, what is their execution order? Or just one? Is there any relevant information? thank you

**Comment by 1798972** [comment]
Can someone answer this question, please?

**Comment by hanshuebner** [comment, marked as answer]
Hi @1798972,

thank you for your question.  Plugins configured on a route are executed before plugins on a service, [explained in the documentation](https://docs.konghq.com/gateway/latest/reference/proxy/#2-plugins-execution).

Kind regards,
Hans

**Reply by 1798972** [reply]
thanks!!!
